puzzle-1/package.json

```diff
@@ -22,6 +22,7 @@
     "eslint": "^7.14.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "jest": "^26.6.3",
```

Comment:
Wasn’t jest already part of the this?

Reply:
No, it was not. Maybe you have jest installed globally in your machine.

Reply:
No, it was not. Maybe you have jest installed globally in your machine.